include/drivers/misr/driver.h

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2021 Wavious LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef _MISR_DRIVER_H_
+#define _MISR_DRIVER_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+
+/**
+ * @brief   MISR Enable Enumeration
+ *
+ * CA       Enable all CA MISRs.
+ * DQ0      Enable all DQ0 MISRs.
+ * DQ1      Enable all DQ1 MISRs.
+ * DQ2      Enable all DQ2 MISRs.
+ * DQ3      Enable all DQ3 MISRs.
+ * ALL      Enable all CA and DQx MISRs.
+ */
+typedef enum misr_en_t
+{
+    MISR_EN_CA  = 0x1,
+    MISR_EN_DQ0 = 0x2,
+    MISR_EN_DQ1 = 0x4,
+    MISR_EN_DQ2 = 0x8,
+    MISR_EN_DQ3 = 0x10,
+    MISR_EN_ALL = 0x1F
+} misr_en_t;
+
+/**
+ * @brief   MISR Enumeration
```

Comment:
Make this a little more descriptive in the description. I would say how these map to different HW MISRs or something. Just more clear about what misr_t is supposed to enumerate.